h2o-core/src/main/java/water/util/BytesStats.java

```diff
@@ -0,0 +1,78 @@
+package water.util;
+
+import java.io.Serializable;
+import water.util.Java7.Objects;
+
+/**
+ * Gathers stats for a byte array.
+ * 
+ * The array has to contain at least one NewLine character,
+ * otherwise it's considered bad, bad.
+ * 
+ * Created by vpatryshev on 3/27/17.
+ */
+public class BytesStats implements Serializable {
+  public final int numLines;
+  public final int maxWidth;
```

Comment:
It means maximum number of bytes. E.g. for UTF8 it's the max number of bytes in UTF8 encoding. UTF8 seems to be our standard for internal strings storage. That's how data are stored currently in our BufferedStrings.